crates/aptos-protos/src/pb/aptos.extractor.v1.serde.rs

```diff
@@ -5622,7 +5619,7 @@ impl serde::Serialize for transaction_payload::Type {
         S: serde::Serializer,
     {
         let variant = match self {
-            Self::ScriptFunctionPayload => "SCRIPT_FUNCTION_PAYLOAD",
+            Self::EntryFunctionPayload => "SCRIPT_FUNCTION_PAYLOAD",
```

Comment:
wow, tricky

Reply:
Yeah, I need to review this PR a couple more times... we have way too many generated files...